docs/UserGuide.md

```diff
@@ -269,12 +279,14 @@ Upon entering quiz mode, the following user input will be recognised:
 
 Upon pressing the `↓ key`, the user will be prompted if they got the answer correct. The user can then press 
 `y` to feedback that they got the correct answer or `n` to feedback that they got an incorrect answer.  
-Once you give a feedback (pressing either `y` or `n`), the review and success frequency for that flashcard will be updated even if you
-quit the quiz prematurely.
+
+The quiz mode works in conjunction with the [statistics](#viewing-statistics-of-flashcard-stats) feature. Quiz attempts are recorded and information about the success frequency can be displayed using the [statistics](#viewing-statistics-of-flashcard-stats) feature.
```

Comment:
Standardize the naming as these hyperlinks don't work. "viewing-the-statistics-of-flashcard-stats" is used in the contents page.